ambry-api/src/test/java/com/github/ambry/account/AccountContainerTest.java

```diff
@@ -164,33 +137,31 @@ public void testConstructAccountAndContainerFromArguments() throws JSONException
   public void testDuplicateContainerName() throws Exception {
     ArrayList<Container> containers = new ArrayList<>();
     // first container with (id=0, name="0")
-    containers.add(
-        new ContainerBuilder((short) 0, "0", refContainerStatuses.get(0), refContainerDescriptions.get(0), refAccountId)
-            .setEncrypted(refContainerEncryptionValues.get(0))
-            .setPreviouslyEncrypted(refContainerPreviousEncryptionValues.get(0))
-            .setCacheable(refContainerCachingValues.get(0))
-            .setBackupEnabled(refContainerBackupEnabledValues.get(0))
-            .setMediaScanDisabled(refContainerMediaScanDisabledValues.get(0))
-            .setReplicationPolicy(refContainerReplicationPolicyValues.get(0))
-            .setTtlRequired(refContainerTtlRequiredValues.get(0))
-            .setContentTypeWhitelistForFilenamesOnDownload(
-                refContainerContentTypeAllowListForFilenamesOnDownloadValues.get(0))
-            .setDeleteTriggerTime(refContainerDeleteTriggerTime.get(0))
-            .build());
+    containers.add(new ContainerBuilder((short) 0, "0", refContainerStatuses.get(0), refContainerDescriptions.get(0),
```

Comment:
This pattern has repeated multiple times.  Would it make be simpler to create a function that takes an Id and name?

Reply:
I have another PR ready to be published. This would be done in that PR.